src/uct/ib/efa/ib_efa_md.c

```diff
@@ -0,0 +1,105 @@
+/**
+ * Copyright (c) 2024, NVIDIA CORPORATION. All rights reserved.
+ *
+ * See file LICENSE for terms.
+ */
+
+#ifdef HAVE_CONFIG_H
+#  include "config.h"
+#endif
+
+#include <uct/ib/efa/ib_efa.h>
+#include <ucs/type/status.h>
+#include <ucs/debug/log.h>
+
+
+static uct_ib_md_ops_t uct_ib_efa_md_ops;
+
+static ucs_status_t uct_ib_efa_md_open(struct ibv_device *ibv_device,
+                                       const uct_ib_md_config_t *md_config,
+                                       uct_ib_md_t **p_md)
+{
+    struct ibv_context *ctx;
+    uct_ib_efadv_md_t *md;
+    struct efadv_device_attr attr;
+    ucs_status_t status;
+    int ret;
+
+    if (md_config->devx == UCS_YES) {
```

Comment:
the config is UCX_IB_MLX5_DEVX
why would it affect EFA?

Reply:
MLX5_DEVX config should not make any difference for EFA

Reply:
removed if block